include/ts/DbgCtl.h

```diff
@@ -90,6 +90,10 @@ class DbgCtl
   static void print(char const *tag, char const *file, char const *function, int line, char const *fmt_str, ...)
     TS_PRINTFLIKE(5, 6);
 
+  // No copying/moving.
+  DbgCtl(DbgCtl const &)            = delete;
```

Comment:
Do we also need to delete `DbgCtl(DbgCtl const &&)` these days?

Reply:
Moving defaults to whatever copying is:  https://godbolt.org/z/cETc71dxr .